pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "torchvision",
     "llvmlite>=0.40.0",
     "umap-learn",
+    "opencv-python",
```

Comment:
Remove this as covered by and clashes with `opencv-python-headless`. If you are having issues with `import cv2` failing, delete your `.venv` folder and run `uv sync --locked`.